cli/exec/flags.go

```diff
@@ -0,0 +1,266 @@
+		Name:   "build-created",
+	},
+	cli.Int64Flag{
+		EnvVar: "WOODPECKER_BUILD_STARTED",
+		Name:   "build-started",
+	},
+	cli.Int64Flag{
+		EnvVar: "WOODPECKER_BUILD_FINISHED",
+		Name:   "build-finished",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_BUILD_STATUS",
+		Name:   "build-status",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_BUILD_EVENT",
+		Name:   "build-event",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_BUILD_LINK",
+		Name:   "build-link",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_BUILD_TARGET",
+		Name:   "build-target",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_COMMIT_SHA",
+		Name:   "commit-sha",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_COMMIT_REF",
+		Name:   "commit-ref",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_COMMIT_REFSPEC",
+		Name:   "commit-refspec",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_COMMIT_BRANCH",
+		Name:   "commit-branch",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_COMMIT_MESSAGE",
+		Name:   "commit-message",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_COMMIT_AUTHOR_NAME",
+		Name:   "commit-author-name",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_COMMIT_AUTHOR_AVATAR",
+		Name:   "commit-author-avatar",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_COMMIT_AUTHOR_EMAIL",
+		Name:   "commit-author-email",
+	},
+	cli.IntFlag{
+		EnvVar: "WOODPECKER_PREV_BUILD_NUMBER",
+		Name:   "prev-build-number",
+	},
+	cli.Int64Flag{
+		EnvVar: "WOODPECKER_PREV_BUILD_CREATED",
+		Name:   "prev-build-created",
+	},
+	cli.Int64Flag{
+		EnvVar: "WOODPECKER_PREV_BUILD_STARTED",
+		Name:   "prev-build-started",
+	},
+	cli.Int64Flag{
+		EnvVar: "WOODPECKER_PREV_BUILD_FINISHED",
+		Name:   "prev-build-finished",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_PREV_BUILD_STATUS",
+		Name:   "prev-build-status",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_PREV_BUILD_EVENT",
+		Name:   "prev-build-event",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_PREV_BUILD_LINK",
+		Name:   "prev-build-link",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_PREV_COMMIT_SHA",
+		Name:   "prev-commit-sha",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_PREV_COMMIT_REF",
+		Name:   "prev-commit-ref",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_PREV_COMMIT_REFSPEC",
+		Name:   "prev-commit-refspec",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_PREV_COMMIT_BRANCH",
+		Name:   "prev-commit-branch",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_PREV_COMMIT_MESSAGE",
+		Name:   "prev-commit-message",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_PREV_COMMIT_AUTHOR_NAME",
+		Name:   "prev-commit-author-name",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_PREV_COMMIT_AUTHOR_AVATAR",
+		Name:   "prev-commit-author-avatar",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_PREV_COMMIT_AUTHOR_EMAIL",
+		Name:   "prev-commit-author-email",
+	},
+	cli.IntFlag{
+		EnvVar: "WOODPECKER_JOB_NUMBER",
+		Name:   "job-number",
+	},
+	cli.StringSliceFlag{
+		EnvVar: "WOODPECKER_ENV",
+		Name:   "env, e",
```

Comment:
I checked the code: Only `env` is used.

```suggestion
		Name:   "env",
```